extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -374,11 +393,39 @@ private function imageListAdditionalAction( $sType, DatabaseBase $oDB, Array $aV
 		] );
 	}
 
+	public function getImageStats() {
+		$stats = [];
+		foreach( $this->image_stats_cache_keys as $key => $cache_key ) {
+			$value = $this->wg->memc->get( $cache_key );
+			if ( !empty( $value ) ) {
+				$stats[$key] = $value;
+			}
+		}
+		return $stats;
+	}
+
+	public function setImageStats ( $new_stats ) {
+		foreach ( $this->image_stats_cache_keys as $key => $cache_key ) {
+			if ( array_key_exists( $key, $new_stats ) ) {
+				$this->wg->memc->set( $cache_key, $new_stats[$key], 60*60 );
+			}
+		}
+	}
+
+	public function updateImageStats ($key, $relative_change ) {
+		if ( array_key_exists( $key, $this->image_stats_cache_keys ) ) {
+			if ( $relative_change > 0 ) {
+				$this->wg->memc->incr( $this->image_stats_cache_keys[$key], $relative_change );
+			} else {
+				$this->wg->memc->decr( $this->image_stats_cache_keys[$key], -$relative_change );
```

Comment:
https://github.com/memcached/memcached/wiki/Commands#incrdecr

You cannot incr/decr with negative values. They accept only positive.